config/internal-config.ts

```diff
@@ -0,0 +1,25 @@
+export const RPC_BASE_URL = 'http://nest:3000';
+export const CHAIN_DEPLOYMENT_ID = "11155111"
```

Comment:
it seems both `RPC_BASE_URL` and `CHAIN_DEPLOYMENT_ID` are currently constant. 

`CHAIN_DEPLOYMENT_ID` will come from the deployment step so we can hold it for now and implement it later but `RPC_BASE_URL` should be changeable through ENV to allow us for local testing and change environments.